samples/dnn/js_face_recognition.html

```diff
@@ -87,104 +86,107 @@
 };
 
 function main() {
-  // Create a camera object.
```

Comment:
I think that we can keep `main` function unchanged but do something like that:
```js
function main() {
  // ...
}

utils.loadOpenCv(() => {
  main();
}
```

and
```html
<body>
```
instead of
```html
<body onload="main()">
```

Reply:
Sounds good! Thanks!